frontend/src/main.ts

```diff
@@ -5,17 +5,17 @@
  */
 
 // Plugins
-import { registerPlugins } from '@/plugins'
+import {registerPlugins} from '@/plugins';
 
 // Components
-import App from './App.vue'
+import App from './App.vue';
 
 // Composables
-import { createApp } from 'vue'
+import {createApp} from 'vue';
 
-const app = createApp(App)
+const app = createApp(App);
 
 // adds router and vuetify
```

Comment:
you removed the main.ts?
Where is the app created and mounted - I don't see it in this PR...

Reply:
no, I didn't. why do you think I did? these changes here are only cosmetical - like removing empty spaces or adding semicolons